core/blockchain/impl/block_tree_impl.cpp

```diff
@@ -111,6 +111,7 @@ namespace kagome::blockchain {
       std::shared_ptr<BlockHeaderRepository> header_repo,
       std::shared_ptr<BlockStorage> storage,
       const primitives::BlockId &last_finalized_block,
+      std::shared_ptr<network::ExtrinsicObserver> author_api_observer,
```

Comment:
[optional]. Maybe call it `extrinsic_observer`? AuthorApiObserver is very specific name, extrinsic_observer is more general

Reply:
Agree, it makes sence